src/usr/local/www/services_dhcp.php

```diff
@@ -1379,6 +1390,13 @@ function build_pooltable() {
 					<tr>
 						<th><?=gettext("Static ARP")?></th>
 						<th><?=gettext("MAC address")?></th>
+<?php
+	if ($got_cid) {
```

Comment:
Please use if (...): / else: / endif; style in cases where HTML content is printed inside the conditional